docs/tutorial/index.rst

```diff
@@ -282,6 +285,270 @@ you will access the build logs,
+Luckily, the ``.readthedocs.yaml`` also allows you to specify
+which requirements to install.
+
+To install the library code of your project,
+go back to editing ``.readthedocs.yaml`` on GitHub and modify it as follows:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+   :emphasize-lines: 3-5
+
+   python:
+     version: "3.9"
+     install:
+       - method: pip
+         path: .
+
+With this change, Read the Docs will install the Python code in the virtual environment
+before starting the Sphinx build, which will finish seamlessly.
+If you go now to the API page of your HTML documentation,
+you will see the ``lumache`` summary!
+
+Enabling PDF and EPUB builds
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sphinx can build several other formats in addition to HTML, such as PDF and EPUB.
+You might want to enable this formats for your project
+so your users can read the documentation offline.
+
+To do so, add this extra content to your ``.readthedocs.yaml``:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+   :emphasize-lines: 4-6
+
+   sphinx:
+     fail_on_warning: true
+
+   formats:
+     - pdf
+     - epub
+
+After this change, PDF and EPUB downloads will be available
+both from the "Downloads" section of the :term:`project home`,
+as well as the :term:`flyout menu`.
+
+.. figure:: /_static/images/tutorial/flyout-downloads.png
+   :align: center
+   :alt: Downloads available from the flyout menu
+
+   Downloads available from the flyout menu
+
+Versioning documentation
+------------------------
+
+.. Explain how to manage versions on RTD: create release branches,
+   activate the corresponding version, browse them in the version selector,
+   selectively build versions
+
+Read the Docs allows you to have :doc:`several versions of your documentation </versions>`,
+in the same way that you have several versions of your code.
+By default, it creates a ``latest`` version
+that points to the default branch of your version control system
+(``main`` in the case of this tutorial),
+and that's why the URLs of your HTML documentation contain the string ``/latest/``.
+
+Creating a new version
+~~~~~~~~~~~~~~~~~~~~~~
+
+Let's say you want to create a ``1.0`` version of your code,
+with a corresponding ``1.0`` version of the documentation.
+For that, first navigate to your GitHub repository, click on the branch selector,
+type :kbd:`1.0.x`, and click on "Create branch: 1.0.x from 'main'"
+(more information `on their documentation`__).
+
+__  https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository
+
+Next, go to your :term:`project home`, click on the "Versions" button,
+and under "Active Versions" you will see two entries:
+
+- The ``latest`` version, pointing to the ``main`` branch, and "Public".
+- A new ``stable`` version, pointing to the ``origin/1.0.x`` branch, and also "Public".
```

Comment:
privacy levels only make sense in .com, in .org everything is public and can't be changed.

Reply:
Right, I was describing what the user sees. But there's no point in mentioning it if it can't be changed.